scss/forms/_quantity-selector.scss

```diff
@@ -0,0 +1,63 @@
+.quantity-selector {
+  width: $quantity-selector-width;
+
+  .form-control {
+    max-width: $quantity-selector-input-max-width;
+    text-align: center;
+    transition: none; // stylelint-disable-line property-disallowed-list
```

Comment:
Can't be use instead `@include transition(none);`?

Reply:
I just checked and '@include transition(none);' works perfectly so we can use it.